packages/atlas/src/components/_overlays/ClaimChannelPaymentsDialog/ClaimChannelPaymentsDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import BN from 'bn.js'
+import { useMemo } from 'react'
+
+import { useFullChannel } from '@/api/hooks/channel'
+import { SvgJoyTokenMonochrome24 } from '@/assets/icons'
+import { Fee } from '@/components/Fee'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Text } from '@/components/Text'
+import { TextWrapper } from '@/components/WidgetTile/WidgetTile.styles'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { BalanceWrapper } from '@/components/_overlays/ClaimChannelPaymentsDialog/ClaimChannelPaymentsDialog.styles'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { hapiBnToTokenNumber } from '@/joystream-lib/utils'
+import { useFee, useSubscribeAccountBalance, useTokenPrice } from '@/providers/joystream/joystream.hooks'
+import { useUser } from '@/providers/user/user.hooks'
+import { formatNumber } from '@/utils/number'
+import { useChannelPayout } from '@/views/studio/MyPaymentsView/PaymentsOverviewTab/PaymentsOverviewTab.hooks'
+
+interface ClaimChannelPaymentsDialogProps {
+  show?: boolean
+  onExit?: () => void
+}
+
+export const ClaimChannelPaymentsDialog = ({ onExit, show }: ClaimChannelPaymentsDialogProps) => {
+  const { channelId } = useUser()
+  const { channel } = useFullChannel(channelId || '')
+  const { availableAward, claimReward, rawHapiAward, isAwardLoading, txParams } = useChannelPayout(onExit)
+  const { fullFee, loading: feeLoading } = useFee('claimRewardTx', txParams)
```

Comment:
I'm pretty sure that fee isn't correctly calculated. When I logged txParams I got `undefined`